editor/components/block-mover/index.js

```diff
@@ -15,14 +14,16 @@ function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast } ) {
 	return (
 		<div className="editor-block-mover">
 			<IconButton
-				className={ classnames( 'editor-block-mover__control', { 'is-disabled': isFirst } ) }
+				className="editor-block-mover__control"
 				onClick={ onMoveUp }
 				icon="arrow-up-alt2"
+				disabled={ !! isFirst }
```

Comment:
Lack of types make it not so obvious, but we should (and currently can) assume the incoming value is already a boolean.